src/main/java/com/google/sps/servlets/UserServlet.java

```diff
@@ -42,6 +52,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       // take the user elements and create a new entity object for it 
       FullEntity userEntity =
         Entity.newBuilder(keyFactory.newKey(id))
+            .set("name", name)
             .set("email", email)
             .set("wishlist", wishlist)
             .set("teachlist", teachlist)
```

Comment:
Store lists as actual lists:
```suggestion
            .set("wishlist", wishlist.split(','))
            .set("teachlist", teachlist.split(','))
```

From the Java docs:
![image](https://user-images.githubusercontent.com/8023233/114496281-75ffc300-9bd4-11eb-8cca-c7e5d9b2e1cc.png)